mava/systems/ippo/networks.py

```diff
@@ -225,7 +255,13 @@ def policy_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
         """
         # Add the observation network and an MLP network.
         policy_network = [
-            hk.nets.MLP(policy_layer_sizes, activation=jax.nn.relu),
+            hk.nets.MLP(
+                policy_layer_sizes,
+                activation=activation_function,
+                w_init=w_init_fn(orthogonal_initialisation, jnp.sqrt(2)),
+                b_init=b_init_fn(orthogonal_initialisation, 0.0),
```

Comment:
I think we can maybe remove the bias init since the default in haiku for the bias is 0 anyway - [link](https://github.com/deepmind/dm-haiku/blob/307cf7dbda64d637ca423cacc9978f0ca19dc8a6/haiku/_src/basic.py#L156).

Reply:
Good catch! Removed it :)